recipes/bioconductor-flowsom/build.sh

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Can you remove the empty lines here?
I think you are not using the latest version of the skeleton generator, which is now part of bioconda-utils. We fixed this bug imho. Sorry for the inconvenience.

Reply:
not a pb!